AEA-1258 Docs Review: Development Beginner (#2234)
fetchai/agents-aea: docs/generic-skills-step-by-step.md

The above code handles an unidentified dialogue by responding to the sender with a `DefaultMessage` containing the appropriate error information.

> The next code block handles `CFP` (call-for-protposal) negotiation messages. Paste the following code below the `_handle_unidentified_dialogue` function:
> 

Reviewer: ```suggestion
The next code block handles `CFP` (call-for-proposal) negotiation messages. Paste the following code below the `_handle_unidentified_dialogue` function:

```

        self._data_for_sale = data_for_sale
```

> In the above code snippet, we initialise the strategy class by trying to read the variables specific to the strategy from a YAML configuration file. If any variable isn't provided, some default values will be used.

Reviewer: ```suggestion
In the above code snippet, we initialise the strategy class by trying to read the variables specific to the strategy from a YAML configuration file. If any variable is not provided, some default values will be used.
```

        )
```

> The various dialogues classes in the above code snippet store dialogues with other AEAs, services and components, (e.g. SOEF search node, Ledgers, Decision maker). They expose useful methods to manipulate these interactions, access previous messages, and enable us to identify possible communications problems between `my_generic_seller` and `my_generic_buyer` AEAs.

Reviewer: ```suggestion
The various dialogues classes in the above code snippet store dialogues with other AEAs, services and components, (e.g. SOEF search node via the `fetchai/soef` connection, ledgers via the `fetchai/ledger` connection, the decision maker). They expose useful methods to manipulate these interactions, access previous messages, and enable us to identify possible communications problems between `my_generic_seller` and `my_generic_buyer` AEAs.
```
